x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/lucene/read/ValuesReader.java

```diff
@@ -36,9 +36,6 @@ public Block[] next() {
         boolean success = false;
         try {
             load(target, offset);
-            if (target[0].getPositionCount() != docs.getPositionCount()) {
```

Comment:
Can we add this check to `ValuesFromManyReader`?

Reply:
yeah. I can move it there. I'll make the in-order version of that work soon too.